src/main/scala/org/squbs/sample/SampleHttpService.scala

```diff
@@ -63,7 +63,7 @@ class SampleHttpService extends RouteDefinition {
   private val chunks =
     path("hello" / Segment / IntNumber) { (who, delay) =>
       onComplete(ActorLookup ? ChunkRequest(who, delay milliseconds)) {
-        case Success(source: Source[PingResponse, Any]) =>
+        case Success(source: Source[_, _]) =>
```

Comment:
hmm.  We could but isn't this change more "scala like" ?  I.e the Source `types` are not strictly necessary for this pattern match.  So type erasure while unfortunate is not a major impediment in scala. With Java though It is necessary.  So I think this is a reasonable deviation from having both templates be "identical"